### PR TITLE
Create delete channel functionality

### PR DESCRIPTION
**Pull request by chrishill93**
Adding create and delete channel functionalities.  Create and Delete channel functions added to app.py.  Utils.py contains functions required for Create and Delete Channel functions.  Test_utils.py created to test utils.py.

### CODE REVIEW

**Comment by chrishill93** [comment]
We will need to create tests for the Create and Delete Channel routes in app.py.  I could also try refactoring these into their own files.  What do you guys think?  Could be an issue for next week.

**Comment by chrishill93** [comment]
I am at a loss here.  The test for utils.py passes, but the test for channel_management is returning an error trying to test the create_channel method.  If I'm understanding it correctly, for some reason the user id isn't being passed into the user_channel table?  I'm going in circles and can't figure out why.

**Comment by chrishill93** [comment]
Solved.  Future reference: don't forget @pytest.fixture when using fixtures...

**Review by Sawneet-24 — APPROVED** [review]
Ran the tests and all 16 passed.

File: project/tests/test_channel_management.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    with app.app_context():
+        db.create_all()
+        yield app
+        db.drop_all()
+
+# Creating a test user
+@pytest.fixture
+def create_test_user():
+    uname = "test_user"
+    test_user = User(username=uname, password="test_password")
```
Comment (jacobtizel) [inline project/tests/test_channel_management.py:23]: Implement password security (don't store in plaintext)

**Pull request merged**